services/api/src/resources/group/opendistroSecurity.ts

```diff
@@ -45,7 +45,7 @@ export const OpendistroSecurityOperations = (
         index_permissions: [
           {
             index_patterns: [],
-            allowed_actions: ['read']
+            allowed_actions: ['read','indices:monitor/settings/get']
```

Comment:
What does this change allow?

Reply:
This is one of the core permissions required to add reporting access to user accounts in ODFE. Without this permissions, it appears that users are unable to select the data needed to create reports.